broker/plugins/kubejobs/plugin.py

```diff
@@ -233,11 +263,34 @@ def errors(self):
             return ()
         return self.rds.lrange("job:errors", 0, -1)
 
+    def persist_state(self):
+        self.persistence_obj.\
+            put(self.app_id, self)
+
+    def synchronize(self):
+        try:
+            current_status = self.k8s.get_job_status(self.app_id)
+            if current_status.active is not None:
+                self.update_application_state("ongoing")
+            else:
+                condition = current_status.conditions[-1].type
+                if condition == 'Complete':
```

Comment:
"Complete" or "Completed"? asking because I never saw the former in k8s before.

Reply:
It's Complete.